firebase.json

```diff
@@ -126,7 +126,10 @@
       { "source": "/events/2016{,/**}", "destination": "https://events.dartlang.org/2016/summit", "type": 301 },
       { "source": "/events{,/**}", "destination": "https://events.dartlang.org", "type": 301 },
       { "source": "/flutter", "destination": "https://flutter.io", "type": 301 },
-      { "source": "/go/null-safety-migration", "destination": "https://github.com/dart-lang/sdk/tree/master/pkg/nnbd_migration#providing-feedback", "type": 301 },
+
+      { "source": "/go/experiments", "destination": "https://dart.dev/tools/experiment-flags", "type": 301 },
```

Comment:
Oops, I just realized that the destination path should start with /, not https://dart.dev/. Not a big deal. I'll fix it later.

Reply:
Ah, since it's redirecting within the same site? Makes sense.